Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.Properties.cs

```diff
@@ -0,0 +1,195 @@
+            nameof(NoSelectionContentTemplate),
+            typeof(DataTemplate),
+            typeof(MasterDetailsView),
+            new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the selected item.
+        /// </summary>
+        /// <returns>The selected item. The default is null.</returns>
+        public object SelectedItem
+        {
+            get { return (object)GetValue(SelectedItemProperty); }
+            set { SetValue(SelectedItemProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the DataTemplate used to display the details.
+        /// </summary>
+        public DataTemplate DetailsTemplate
+        {
+            get { return (DataTemplate)GetValue(DetailsTemplateProperty); }
+            set { SetValue(DetailsTemplateProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the Brush to apply to the background of the list area of the control.
+        /// </summary>
+        /// <returns>The Brush to apply to the background of the list area of the control.</returns>
+        public Brush MasterPaneBackground
+        {
+            get { return (Brush)GetValue(MasterPaneBackgroundProperty); }
+            set { SetValue(MasterPaneBackgroundProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the content for the master pane's header
+        /// </summary>
+        /// <returns>
+        /// The content of the master pane's header. The default is null.
+        /// </returns>
+        public object MasterHeader
+        {
+            get { return (object)GetValue(MasterHeaderProperty); }
+            set { SetValue(MasterHeaderProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the DataTemplate used to display the content of the master pane's header.
+        /// </summary>
+        /// <returns>
+        /// The template that specifies the visualization of the master pane header object. The default is null.
+        /// </returns>
+        public DataTemplate MasterHeaderTemplate
+        {
+            get { return (DataTemplate)GetValue(MasterHeaderTemplateProperty); }
+            set { SetValue(MasterHeaderTemplateProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the width of the master pane when the view is expanded.
+        /// </summary>
+        /// <returns>
+        /// The width of the SplitView pane when it's fully expanded. The default is 320
+        /// device-independent pixel (DIP).
+        /// </returns>
+        public double MasterPaneWidth
+        {
+            get { return (double)GetValue(MasterPaneWidthProperty); }
+            set { SetValue(MasterPaneWidthProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the content to dsiplay when there is no item selected in the master list.
+        /// </summary>
+        public object NoSelectionContent
```

Comment:
I think this property should be demonstrated in the sample app (like a centered text : "Please select an item")